src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -18,15 +18,19 @@ import NavPageLink from './NavPageLink';
 import DrawerHeader from '../../components/DrawerHeader';
 import { ChevronLeft, ChevronRight } from '@mui/icons-material';
 import NERDrawer from '../../components/NERDrawer';
-import { LayoutProps } from '../LayoutProps';
+import { GridMenuIcon } from '@mui/x-data-grid';
+import { useState } from 'react';
+import NavUserMenu from '../PageTitle/NavUserMenu';
 
-export interface SideBarProps extends LayoutProps {
-  handleDrawerClose: () => void;
+interface SidebarProps {
+  defaultOpen?: boolean;
 }
 
-const Sidebar: React.FC<SideBarProps> = ({ open, handleDrawerClose }) => {
+const Sidebar: React.FC<SidebarProps> = (defaultOpen?) => {
```

Comment:
you can do defaultOpen = false here (no ?) instead of setting it to false by default in the useState. It'll do the same thing

Reply:
sorry actually like this, I think what I suggested was causing a typescript error: 
<img width="342" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/5652c2f3-b520-426e-9921-c330f7488fce">